Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabUiHandler.cpp

```diff
@@ -306,41 +306,37 @@ namespace AzToolsFramework
         {
             // Only show the close icon if the prefab is expanded.
             // This allows the prefab container to be opened if it was collapsed during propagation.
-            if (!isExpanded)
+            if (isExpanded)
```

Comment:
I think you can shorten this by combining the two ifs
if (m_prefabFocusPublicInterface->IsOwningPrefabBeingFocused(entityId)) && isExpanded)